package.json

```diff
@@ -5,15 +5,19 @@
   "main": "dist/server.bundle.js",
   "scripts": {
     "bootstrap": "shx rm -rf dist && shx mkdir dist",
-    "start": "npm run build && run-p server watch:**",
+    "start": "npm run build:dev && run-p server watch:**",
     "build": "run-s bootstrap build:**",
-    "heroku-postbuild": "echo Skip builds on Heroku",
+    "heroku-postbuild": "npm run build:prod",
+    "build:prod": "run-s bootstrap build-client:** build-js:prod:**",
```

Comment:
Webpack v4 introduced a new option `mode` and it has to be explicitly set (e.g., we can utilize variable like `process.env.NODE_ENV`.) 

I used this [guide](https://webpack.js.org/guides/production/#setup) to break our existing webpack config into "prod" and "dev" version.